### DISCUSSION TITLE
Use of FreeMonitor to set pre-defined accessories value

### DISCUSSION THREAD

**Discussion by Tellicious** [RavenSystem/esp-homekit-devices#2080] [MEPLHAA Scripts]
Can FreeMonitor be used to set the value of other pre-defined accessories such as Temperature and Humidity / Light Sensor?

I'm planning to use an I2C CO2 sensor that can also read temperature and humidity: can I push those values to an empty Temperature and Humidity sensor?

**Comment by RavenSystem** [comment]
Yes, using "tg":[] https://github.com/RavenSystem/esp-homekit-devices/wiki/free-monitor#target-homekit-characteristic
